include/dfrn.php

```diff
@@ -113,6 +112,7 @@ public static function feed($dfrn_id, $owner_nick, $last_update, $direction = 0,
 		);
 
 		if (! dbm::is_result($r)) {
+			logger(sprintf('No contact found for nickname=%d', $owner_nick), LOGGER_WARNING);
```

Comment:
See here for a list: https://github.com/friendica/friendica/blob/develop/boot.php#L114-L118